api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -507,6 +507,18 @@ message Features {
   // Policy enables the Teleport Policy feature set.
   // At the time of writing, this includes Teleport Access Graph (TAG).
   PolicyFeature Policy = 28 [(gogoproto.jsontag) = "policy,omitempty"];
+  // Questionnaire indicates whether cluster users should get an onboarding questionnaire
+  bool Questionnaire = 29;
```

Comment:
Obligatory "bool vs string vs enum" links.

* https://cloud.google.com/apis/design/design_patterns#bool_vs_enum_vs_string
* https://protobuf.dev/programming-guides/api/#never-use-booleans-for-two-states
* https://github.com/gravitational/teleport/blob/master/rfd/0153-resource-guidelines.md#defining-a-resource

This one is already full of bools so feel free to disregard.

Reply:
Thanks for the resources!

I've updated the Support flag to be an enum, as this can have more tiers in the future.

I kept booleans for the flags that just enable or disable a feature since this is the current pattern.

I'm on the fence on `Questionnaire`, as I can see we have more questionnaires in the future, but they could coexist, so having an enum might not be enough, we might need multiple fields like `OnboardingQuestionnaire`, `OffboardingQuestionnaire`, `OtherQuestionnaire`, etc. I think we can keep `bool Questionnaire` for now, since renaming it wouldn't be a problem.